assignment2/task1.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: UTF-8 -*-
+
+
+class AncestorsFinder(object):
+    def __init__(self, graph):
+        self.graph = graph
+        self.root = 0
+        self.ancestors = None
+
+    def find(self, value):
+        self.ancestors = []
+        self._dfs(self.root, -1, value)
+        return self.ancestors
+
+    def _dfs(self, ver, prev, value):
```

Comment:
Can you please add a comment to this method and explain all the input parameters?